### ISSUE TITLE
[deeplab] [color palette] Mapping between RGB values and class numbers

### ISSUE THREAD

**Issue by lydialixia** [tensorflow/models#4195]
Hi, I'm training deeplab V3+ with my own dataset, based on Xception checkpoint that's pretrained on ImageNet. 

I have totally 5 classes including background. My labels are 1-channel png that each pixel is in [0,4]. After training, the results on test image are, however, 3-channel images (RGB: [0,0,0], [0,0,128], [0,128,0], [128,0,0], [128,128,0]). How can I know which RGB value corresponds to which class number? In other words, how to get the mapping between RGB values and class numbers?

Thanks a lot!

**Comment by aquariusjay** [comment]
You could set the flag, [also_save_raw_predictions](https://github.com/tensorflow/models/blob/master/research/deeplab/vis.py#L86), to True so that the raw predictions (with value = [0, 1, ..., num_classes-1]) are also saved.
